tests/test_primitives/test_mux.py

```diff
@@ -174,3 +176,33 @@ class test_mux_operator_int(m.Circuit):
     tester.compile_and_run("verilator", skip_compile=True,
                            directory=os.path.join(os.path.dirname(__file__),
                                                   "build"))
+
+
+def test_mux_operator_tuple():
+    class test_mux_operator_tuple(m.Circuit):
+        io = m.IO(S=m.In(m.Bit), O0=m.Out(m.Bit), O1=m.Out(m.Bits[2]),
+                  O2=m.Out(m.Bit))
+        O0, O1, O2 = m.mux([
```

Comment:
does this unpack `m.Tuple` type automatically? or do we need to call `.values()` on the output first?

Reply:
It does by using the Iterable interface (which tuple values provide)